ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -60,7 +60,8 @@ class azure_connection {
     int put_object(const std::string &object_key, const std::string &file_path) const;
     int delete_object(const std::string &object_key) const;
     int read_object(const std::string &object_key, int64_t offset, size_t len, void *buf) const;
-    int object_exists(const std::string &object_key, bool &exists) const;
+    int object_exists(const std::string &object_key, bool &exists, size_t &object_size) const;
+    int object_size(const std::string &object_key) const;
```

Comment:
i don't think we use this function anymore

Reply:
yep mb forgot to clean this up, ty